code/modules/nifsoft/nifsoft.dm

```diff
@@ -377,3 +377,32 @@
 	..()
 	for(var/i = 0 to 7)
 		new /obj/item/weapon/disk/nifsoft/mining(src)
+
+// Pilot Disk //
+/obj/item/weapon/disk/nifsoft/pilot
+	name = "	"
```

Comment:
You did not give it a name. Between the quotation marks, add NIFsoft Uploader -  Pilot